cdap-elastic/src/main/java/io/cdap/cdap/metadata/elastic/ElasticsearchMetadataStorage.java

```diff
@@ -805,10 +831,15 @@ private void setRefreshPolicy(WriteRequest<?> request, MutationOptions options)
       // scroll had expired, we have to search again
       return doSearch(createRequestFromCursor(request, cursor));
     }
-    SearchHits hits = searchResponse.getHits();
+    return createSearchResponse(request, searchResponse, computeCursor(searchResponse, cursor),
+                                cursor.getOffset(), cursor.getLimit());
+  }
+
+  private io.cdap.cdap.spi.metadata.SearchResponse createSearchResponse(SearchRequest request, SearchResponse response,
```

Comment:
instead of returning io.cdap.cdap.spi.metadata.SearchResponse, can we just return SearchResponse by import the pkg.

Reply:
No. There is another `SearchResponse` class from ElasticSearch being imported.